lib/DBDefs/Default.pm

```diff
@@ -136,6 +136,9 @@ sub CANONICAL_SERVER          { "https://musicbrainz.org" }
 # The server used to link to CritiqueBrainz users and reviews.
 sub CRITIQUEBRAINZ_SERVER     { "https://critiquebrainz.org" }
 
+# The URL where static resources are located, excluding the trailing slash.
+sub STATIC_RESOURCES_LOCATION { 'http://localhost:5000/static' }
```

Comment:
Hmm, I’m not sure about that. Perhaps it’s just that the images need `build/` added to their path? Adding `build` here in the config will break the JS, in any case, because it will be searched for in `build/build`.